src/screens/settings/settings.tsx

```diff
@@ -128,7 +128,7 @@ export class SettingsScreenComponent extends React.Component<
                             <Switch
                                 onValueChange={async () => {
                                     try {
-                                        if (this.props.touchID) {
+                                        if (this.props.biometricActive) {
```

Comment:
is this still necessary since when you delete all wallets this settings is reset to false?